server/src/test/java/org/apache/druid/segment/loading/StorageLocationSelectorStrategyTest.java

```diff
@@ -256,4 +269,75 @@ public void testMostAvailableSizeLocationSelectorStrategy() throws Exception
     Assert.assertEquals("The next element of the iterator should point to path local_storage_folder_1",
         localStorageFolder1, loc3.getPath());
   }
+
+  @Test
+  public void testDefaultSelectorStrategyConfig() throws Exception
```

Comment:
In the newly added test cases, the `throw Exception` statement needs to be removed to pass the travis CI.

Reply:
`throws` declaration is useless here, maybe is introduced by code copy-paste. Been fixed.